tt_metal/CMakeLists.txt

```diff
@@ -14,21 +14,31 @@ set(TT_METAL_OBJECTS
     $<TARGET_OBJECTS:llrt>
     $<TARGET_OBJECTS:impl>
     $<TARGET_OBJECTS:detail>
-    $<TARGET_OBJECTS:tracy_o>
 )
 
-if(TT_METAL_CREATE_STATIC_LIB)
+if($ENV{TT_METAL_CREATE_STATIC_LIB})
```

Comment:
is this the best way to control static vs. dynamic lib? I thought CMake had a pre-built variable for this

Reply:
now using BUILD_SHARED_LIBS with default building shared libs